packages/cache/src/options.ts

```diff
@@ -0,0 +1,17 @@
+/**
+ * Options to control cache upload
+ */
+export interface UploadOptions {
+  /**
+   * Number of parallel cache upload
+   *
+   * @default 4
+   */
+  uploadConcurrency?: number
+  /**
+   * Maximum chunk size for cache upload
```

Comment:
```suggestion
   * Maximum chunk size in bytes for cache upload
```

Let's clarify bytes here, it might be confusing if users think the default is `32` instead of 32 * 1024 * 1024